R/plot_RLum.R

```diff
@@ -126,25 +124,15 @@ plot_RLum<- function(
         mtext <- rep(list(...)$mtext, length.out = length(object))
```

Comment:
What about this one?

Reply:
There it's not creating a list. I tried to replicate the current behaviour, but I could convert that case too.

Reply:
A clearer explanation: in the `main` case, we were sometimes creating a list (`main.list`) and depending on whether that `main.list` was created we were passing the corresponding element to `RLum.dispatcher()`. I decided to convert it always to a list so we could simplify the dispatch code (see the next hunk, lines 129-142). For `mtext` we were not creating a list in the first place, so I left it untouched.